docs/DeveloperGuide.md

```diff
@@ -439,87 +439,748 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+3. TaskHub assigns the employee to the project and displays the updated list of employees and projects.
+4. TaskHub shows that the employee has been assigned to the project successfully.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. User enters an invalid command.
+  * 2a1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2b. User does not enter a required field.
+  * 2b1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2c. User enters an invalid value for a field.
+  * 2c1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2d. User enters multiple values for a single-valued field.
+  * 2d1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2e. User enters an employee that is already assigned to the project.
+  * 2e1. TaskHub shows success message but there is no effect on the project.
+
+    Use case ends.
+
+**Use case: UC21 - Un-assign an employee from a project**
+
+**MSS**
+
+1. User lists all employees and projects (UC19).
+2. User requests to un-assign an employee from a project.
+3. TaskHub un-assigns the employee from the project and displays the updated list of employees and projects.
+4. TaskHub shows that the employee has been un-assigned from the project successfully.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. User enters an invalid command.
+  * 2a1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2b. User does not enter a required field.
+  * 2b1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2c. User enters an invalid value for a field.
+  * 2c1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2d. User enters multiple values for a single-valued field.
+  * 2d1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2e. User enters an employee that is not assigned to the project.
+  * 2e1. TaskHub shows an error message.
+
+    Use case ends.
+
+**Use case: UC22 - Assign an employee to a task in a project**
+
+**MSS**
+
+1. User lists all employees and projects (UC19).
+2. User requests to assign an employee to a task in a project.
+3. TaskHub assigns the employee to the task in the project and displays the updated list of employees and projects.
+4. TaskHub shows that the employee has been assigned to the task in the project successfully.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. User enters an invalid command.
+  * 2a1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2b. User does not enter a required field.
+  * 2b1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2c. User enters an invalid value for a field.
+  * 2c1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2d. User enters multiple values for a field.
+  * 2d1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2e. User enters an employee that is already assigned to the task in the project.
+  * 2e1. TaskHub shows success message but there is no effect on the project or task.
+
+    Use case ends.
```

Comment:
Maybe this could be Use case resumes from step 4?

Reply:
Sure, I have changed it.